load_testing/common_flows/flow_helper.py

```diff
@@ -83,7 +83,13 @@ def authenticity_token(response, index=0):
 def querystring_value(url, key):
     # Get a querystring value from a url
     parsed = urlparse(url)
-    return parse_qs(parsed.query)[key][0]
+    try:
+        return parse_qs(parsed.query)[key][0]
+    except:
```

Comment:
Can define the specific exception class to catch here?

Reply:
Thanks, that's a great point, I forgot the specific class but it's not hard to find and integrate.

Reply:
This has been done and bonus, I stumbled upon locust's native logging facility so we can start logging errors to logs rather than stdout/err.